notebooks/243-named-entity-recognition/243-named-entity-recognition.ipynb

```diff
@@ -0,0 +1,260 @@
+{
```

Comment:
<p>lease remove additional colon in the end.</p><p> </p><blockquote>To handle the NNCF quantization process we use class <a href="https://huggingface.co/docs/optimum/intel/reference_ov#optimum.intel.OVQuantizer" target="_blank">OVQuantizer</a>. The quantization with Hugging Face Optimum Intel API contains the next steps:</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
removed